examples/MediaServices/scale_encoding_units.php

```diff
@@ -22,17 +22,14 @@
  *
  * @link      https://github.com/windowsazure/azure-sdk-for-php
  */
-require_once 'vendor/autoload.php';
+require_once __DIR__.'/../vendor/autoload.php';
 
 use WindowsAzure\Common\ServicesBuilder;
 use WindowsAzure\Common\Internal\MediaServicesSettings;
 use WindowsAzure\MediaServices\Models\EncodingReservedUnitType;
 
-// Settings
-date_default_timezone_set('America/Los_Angeles');
-
-$account = '<your media services account name>';
-$secret = '<your media services account key>';
+// read user settings from config
+include_once 'userconfig.php';
```

Comment:
I am aware of the difference of require and include. There is really no difference in this case because it is not doing any thing that cause errors, and this example is showcasing the Media Services APIs and teaching people how to write solid code.